src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -6627,7 +6627,7 @@ namespace HeatBalanceManager {
         for (SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; ++SurfNum) {
             print(state.files.shade, "{},", state.dataSurface->Surface(SurfNum).Name);
         }
-        print(state.files.shade, "()\n");
+        print(state.files.shade, "\n");
```

Comment:
The fix for the export is simple, remove these parentheses

Reply:
Super simple and straightforward.  Trying to trace this change back became tedious and I gave up.  Anyway, it's an obvious fix, thanks.

Reply:
It was there in the gio days, except there was a refactor there too.

It was broken starting at 9.1.0, It's this commit: https://github.com/NREL/EnergyPlus/commit/39ffc4329428e1117dc50057efdb1b9f85f8f8a5